tests/conftest.py

```diff
@@ -25,38 +25,99 @@
 
 import pytest
 import yaml
+from _pytest.config import Config
+from _pytest.config.argparsing import Parser
+from _pytest.nodes import Item
```

Comment:
I'm not sure about importing from `_pytest` for typing purposes. I usually try to avoid it under the assumption that it might be an unstable interface. I hate that they don't expose these on `pytest`.
@wouterdb any opinion on this?